timescaledb/getting-started/compress-data.md

```diff
@@ -81,91 +88,41 @@ SELECT * FROM timescaledb_information.job_stats;
-  pg_size_pretty(after_compression_total_bytes) as "after compression"
-  FROM hypertable_compression_stats('weather_metrics');
+SELECT compress_chunk(i)
+FROM show_chunks('stocks_real_time', older_than => INTERVAL ' 2 weeks') i;
 ```
 
-## Benefits of compression
-
-**Disk space savings**
-
-Compression allows you to you enjoy disk space savings, so you can store more
-data in a fixed amount of disk space than you would in other databases (for
-example, [TimescaleDB uses 10% of the disk space to store the same number of
-time-series metrics as
-MongoDB][mongodb-comparison]).
-
-This is especially beneficial when backups and high-availability replicas are
-taken into account, as you'd save disk space and storage costs on all databases.
-
-**Better query performance**
+## Verify your compression
 
-In addition to saving storage space and costs, compressing data might increase
-query performance on certain kinds of queries. Compressed data tends to be older
-data and older data tends to have different query patterns than recent data.
-
-**Newer data tends to be queried in a shallow and wide fashion**. In this case,
-shallow refers to the length of time and wide refers to the range of columns
-queried. These are often debugging or "whole system" queries. For example, "Show
-me all the metrics for all cities in the last 2 days." In this case the
-uncompressed, row-based format that is native to PostgreSQL gives us the best
-query performance.
-
-**Older data tends to be queried in a deep and narrow fashion.** In this case,
-deep refers to the length of time and narrow refers to the range of columns
-queried. As data begins to age, queries tend to become more analytical in nature
-and involve fewer columns. For example, "Show me the average annual temperature
-for city A in the past 20 years." This type of query greatly benefits from the
-compressed, columnar format.
-
-TimescaleDB's compression design allows you to get the best of both worlds:
-recent data is ingested in an uncompressed, row format for efficient shallow and
-wide queries, and then automatically converted to a compressed, columnar format
-after it ages and is most often queried using deep and narrow queries.
-
-Here's an example of a deep and narrow query on the compressed data. It
-calculates the average temperature for New York City for all years in the
-dataset before 2010. Data for these years is compressed, since you compressed all
-data older than 10 years with either the policy or the manual compression method
-above.
+Use this query to view the size of your compressed chunks before and after applying compression:
 
 ```sql
--- Deep and narrow query on compressed data
-SELECT avg(temp_c) FROM weather_metrics
-WHERE city_name LIKE 'New York'
-AND time < '2010-01-01';
+SELECT pg_size_pretty(before_compression_total_bytes) as "before compression",
+  pg_size_pretty(after_compression_total_bytes) as "after compression"
+  FROM hypertable_compression_stats('stocks_real_time');
```

Comment:
Example output would be helpful (realizing that the sizes will be slightly different.